plugins/aac-plantuml/aac-plantuml.yaml

```diff
@@ -6,15 +6,12 @@ model:
       type: command
       description: Converts an AaC model to Plant ULM component diagram
       input:
-        - name: archFile
-          type: file
         - name: parsed_model
           type: map
       acceptance:
         - scenario: Output PlantUML component diagram for valid architecture.
           given:
```

Comment:
I think there's value in including the archFile.  This is a somewhat generic interface that has to be useful to any plugin developer.  For example, sometimes I just like to have the archFile available to provide better error messages / logging.